Gemfile

```diff
@@ -68,7 +68,7 @@ gem 'oj_mimic_json'
 # might need this
 #gem 'activeuuid', '>= 0.5.0'
 
-gem 'snorlax'
+gem 'snorlax', '~> 0.1.3'
```

Comment:
This updates the gem to use

`if current_user.is_logged_in? then (thing) else (other thing)`

instead of

`if current_user then (thing) else (other thing)`

which naturally doesn't work with the null user pattern